content/releases/cloudnative-pg-1-26.0-released/index.md

```diff
@@ -0,0 +1,177 @@
+
+- CloudNativePG 1.26.0, featuring major enhancements and new capabilities
+- CloudNativePG 1.25.2, a maintenance update for the 1.25.x series
+- CloudNativePG 1.24.4, the final patch for the now unsupported 1.24.x series
+
+This is our first release since CloudNativePG officially entered the
+[CNCF Sandbox](https://www.cncf.io/sandbox-projects/), a major milestone that
+reinforces our commitment to sustainable, community-driven innovation.
+
+---
+
+## What’s New in CloudNativePG 1.26.0?
+
+### Declarative Offline In-Place Major Upgrades of PostgreSQL
+
+You can now declaratively trigger
+[offline in-place major upgrades](https://cloudnative-pg.io/documentation/1.26/postgres_upgrades/)
+of PostgreSQL by simply updating the container image to a newer major version.
+The cluster shuts down safely, and `pg_upgrade` performs the upgrade, ensuring
+consistency. This long-awaited feature simplifies major upgrades while
+maintaining declarative workflows.
+
+### Enhanced Startup and Readiness Probes for Replicas
+
+We've enhanced [startup and readiness probes](https://cloudnative-pg.io/documentation/1.26/instance_manager/)
+for replicas, enabling both startup and readiness to be gated by replication
+lag. This means you can restrict promotion to only lag-free, synchronous
+replicas—greatly improving your high availability posture.
+
+### Declarative Management of Extensions and Schemas
+
+The `Database` resource now supports the declarative creation of:
+
+- [PostgreSQL extensions](https://cloudnative-pg.io/documentation/1.26/declarative_database_management/#managing-extensions-in-a-database)
+- [`SCHEMA` objects](https://cloudnative-pg.io/documentation/1.26/declarative_database_management/#managing-schemas-in-a-database)
+
+This makes it easier to manage consistent database configurations across
+environments.
+
+---
+
+## Important Changes
+
+### Barman Cloud Deprecation Begins
+
+With the 1.26 release, the **deprecation period** for in-tree Barman Cloud
+support officially begins. While it remains fully functional in 1.26, we
+**strongly encourage** users to begin planning the migration to the
+[Barman Cloud Plugin](https://cloudnative-pg.io/plugin-barman-cloud/)
+as early as possible and to adopt it for all new deployments.
+To help with this, we’ve published a comprehensive
+[migration guide](https://cloudnative-pg.io/plugin-barman-cloud/docs/migration/).
+
+In CloudNativePG 1.28, Barman Cloud will be fully removed from CloudNativePG’s
+core. You have until then to complete your migration.
+
+This marks a significant milestone in CloudNativePG’s evolution—the culmination
+of a multi-year effort that introduced CNPG-I, our extensible plugin interface.
+It is a crucial step toward making CloudNativePG a backup-agnostic solution
+while also enabling leaner operand images by removing the need to bundle Barman
+Cloud directly. It also paves the way for future plugin support for volume
+snapshot backups and restores.
+
```

Comment:
It's a lot of "also". I'd remove the last one and make it "It paves the way for future plugin support for volume snapshot backups and restores"